### ISSUE TITLE
Feature Request: In the forum helpers list, make "Ocular status" show a tooltip when clicked or hovered upon

### ISSUE THREAD

**Issue by penguinmoose** [theforumhelpers/theforumhelpers.github.io#337]
**Is your feature request related to a problem?**
Not really. In the list of forum helpers, on the top, where it says "Below is a list of current ... ... biography, and **Ocular status**", "Ocular status" links to the homepage of Ocular. Instead, I think it would be more helpful if we make hovering or clicking on it show a tooltip that describes what an Ocular status is (the tooltip can have a link to Ocular). If I were a user that did not know what an Ocular status is, going to Ocular's homepage wouldn't help me. 

**Describe alternatives you've considered**
- Small adjustment: the tooltip will only show if hovered upon for a few seconds or if clicked. This will avoid showing the tooltip when the user is just moving their cursor across the page.
- Make a star or something like a superscript number next to "Ocular status" and the bottom of the page will have a description of what an ocular status is.
- Make a tiny circle question mark next to "Ocular status" and a description or tooltip will show if that question mark is clicked. This is also sometimes done on other websites instead of direct tooltips.

**Additional context**
None

**Comment by Accio1** [comment]
Perhaps adding a title attribute of "an ocular status is a short message and favourite colour, which is displayed across ocular and other sites" and changing the link to https://ocular.jeffalo.net/docs/about#what-is-an-ocular-status would be more helpful?

**Comment by penguinmoose** [comment]
@Accio1 I think that would be even better.

**Comment by penguinmoose** [comment]
@Accio1 I put your suggestion in alternatives. Also, I'm not a huge fan of long titles (the attribute), so maybe we can shorten it to something like "an ocular status is a short message and color set using ocular.jeffalo.net" or something like that?

**Comment by Accio1** [comment]
Is the title attribute even required if we change the link to the one mentioned above? If not then you can go ahead and submit a PR to change the link @penguinmoose

**Comment by penguinmoose** [comment]
@Accio1 True. Changing the link is good enough. I’ll make a PR when I have time.
